fortnite_api/enums.py

```diff
@@ -47,7 +49,133 @@
 )
 
 
-class KeyFormat(enum.Enum):
+E = TypeVar('E', bound='Enum')
+
+
+def _create_value_cls(name: str, comparable: bool):
+    # All the type ignores here are due to the type checker being unable to recognise
+    # Runtime type creation without exploding.
+    cls = namedtuple('_EnumValue_' + name, 'name value')
```

Comment:
I'd make this a bit more clear haha,

```suggestion
    # Pyright cannot statically recognize the runtime type creation. All of the following
    # type ignores in this function are a result of this.
    cls = namedtuple('_EnumValue_' + name, 'name value')
```